### PR TITLE
Update Slovak translation to 8.4.3

### PR DESCRIPTION
**Pull request by kubalav**

### CODE REVIEW

**Comment by donho** [comment]
@kubalav 
You should add the commit into the old PR which I just merged.
Please solve the conflict.

**Comment by kubalav** [comment]
> @kubalav You should add the commit into the old PR which I just merged instead of creating the new one. Please solve the conflict.

I'm sorry. I overlooked existing PR. I use to add commit to existing PR, but I didn't see it and I create new one.

File: PowerEditor/installer/nativeLang/slovak.xml [hunk 1]
```diff
@@ -1048,6 +1048,7 @@ Zadať môžete viacero značiek pomocou medzery na oddelenie rôznych čísel."
                     <Item id="6903" name="Dialógové okno Hľadať zostáva otvorené po vyhľadaní s výstupom do okna s výsledkami hľadania"/>
                     <Item id="6904" name="Potvrdiť Nahradiť všetko vo všetkých otvorených dokumentoch"/>
                     <Item id="6905" name="Nahradiť: Neprechádzať na nasledujúci výskyt"/>
+                    <Item id="6906" name="Okno s výsledkami hľadania: zobraziť len jednu položku na nájdený riadok (neplatí pre režim sprava doľava)"/>
```
Comment (donho) [inline PowerEditor/installer/nativeLang/slovak.xml:1051]: "(not applied to RTL mode)" has been removed in this entry, see: https://github.com/notepad-plus-plus/notepad-plus-plus/commit/059bca3315dbd71fbf235c8bc87eaacbd87a3cec
Sorry for the inconvenience.

Comment (kubalav) [reply PowerEditor/installer/nativeLang/slovak.xml]: Done.

**Pull request closed without merging**